integration_test/agents/agents.go

```diff
@@ -184,13 +186,21 @@ func RunOpsAgentDiagnostics(ctx context.Context, logger *logging.DirectoryLogger
 		gce.SyslogLocation(vm.Platform),
 		"/var/log/google-cloud-ops-agent/subagents/logging-module.log",
 		"/var/log/google-cloud-ops-agent/subagents/metrics-module.log",
-		"/run/google-cloud-ops-agent-fluent-bit/fluent_bit_main.conf",
-		"/run/google-cloud-ops-agent-fluent-bit/fluent_bit_parser.conf",
 		"/run/google-cloud-ops-agent-opentelemetry-collector/otel.yaml",
 	} {
 		_, basename := path.Split(log)
 		gce.RunRemotely(ctx, logger.ToFile(basename+txtSuffix), vm, "", "sudo cat "+log)
 	}
+
+	// Capture all files beneath /run/google-cloud-ops-agent-fluent-bit
+	fluentBitDir := "/run/google-cloud-ops-agent-fluent-bit"
+	output, err := gce.RunRemotely(ctx, logger.ToMainLog(), vm, "", fmt.Sprintf("ls '%s'", fluentBitDir))
```

Comment:
this doesn't handle subdirectories, probably should be a recursive `ls` or a `find` command

Reply:
fixed